### PR TITLE
[xdl] Allow deprecating universal modules

### PR DESCRIPTION
**Pull request by sjchmiela**

### CODE REVIEW

File: packages/xdl/src/modules/Modules.js [hunk 1]
```diff
@@ -39,19 +44,23 @@ export function getVersionableModulesForPlatform(platform: Platform): Array<Modu
...
+    moduleConfig =>
+      !!moduleConfig.libName && doesVersionSatisfy(sdkVersion, moduleConfig.sdkVersions)
+  );
 }
 
-export function getPublishableModules(): Array<ModuleConfig> {
-  return expoSdkUniversalModulesConfigs.filter(moduleConfig => !!moduleConfig.libName);
+function doesVersionSatisfy(version: string, versionRequirement: string): boolean {
+  if (version === 'UNVERSIONED') {
+    return semver.satisfies(HIGHEST_KNOWN_VERSION, versionRequirement);
```
Comment (tsapeta) [inline packages/xdl/src/modules/Modules.js]: where is `HIGHEST_KNOWN_VERSION` defined?

Comment (sjchmiela) [reply packages/xdl/src/modules/Modules.js]: whoops, I removed it when refactoring 😅

what would you say to removing handling `UNVERSIONED` altogether?

Comment (tsapeta) [reply packages/xdl/src/modules/Modules.js]: iirc this was required for sth to work, but I don't remember for what 😅 maybe you can try 😂

Comment (sjchmiela) [reply packages/xdl/src/modules/Modules.js]: ah, for building ios shell apps, nvm

**Pull request merged**